src/commands/dev/dev.js

```diff
@@ -522,11 +559,33 @@ const createDevCommand = (program) => {
         .hideHelp(),
     )
     .addOption(new Option('--graph', 'enable Netlify Graph support').hideHelp())
+    .addOption(
+      new Option(
+        '-e, --edgeInspect [address]',
+        'enable the V8 Inspector Protocol for Edge Functions, with an optional address in the host:port format',
+      ).conflicts('edgeInspectBrk'),
+    )
+    .addOption(
+      new Option(
+        '-E, --edgeInspectBrk [address]',
+        'enable the V8 Inspector Protocol for Edge Functions and pause execution on the first line of code, with an optional address in the host:port format',
+      ).conflicts('edgeInspect'),
+    )
     .addExamples([
       'netlify dev',
       'netlify dev -d public',
       'netlify dev -c "hugo server -w" --targetPort 1313',
       'netlify dev --graph',
+      'netlify dev -e',
+      'netlify dev -e 127.0.0.1:9229',
+      'netlify dev -e127.0.0.1:9229',
+      'netlify dev --edgeInspect',
+      'netlify dev --edgeInspect=127.0.0.1:9229',
+      'netlify dev -E',
+      'netlify dev -E 127.0.0.1:9229',
+      'netlify dev -E127.0.0.1:9229',
+      'netlify dev --edgeInspectBrk',
+      'netlify dev --edgeInspectBrk=127.0.0.1:9229',
```

Comment:
[dust] I don't think we need to add an example for every single permutation, otherwise we'll end up with a huge list in the help command. How do you feel about keeping just the ones below?

```suggestion
      'netlify dev --edgeInspect',
      'netlify dev --edgeInspect=127.0.0.1:9229',
      'netlify dev --edgeInspectBrk',
      'netlify dev --edgeInspectBrk=127.0.0.1:9229',
```

Reply:
I added the short form ones just in case someone uses the `=` sign with `-e` or `-E` and it fails, they see an example of how go use them with the `--help` flag. This is because something like `-e=127.0.0.1:1234` gives us `edgeInspect: '=127.0.0.1:3030'` because short forms don't support the equal sign causing it to fail.

Reply:
You can do this by specifying a custom parser. See https://github.com/tj/commander.js/#custom-option-processing.

Reply:
@eduardoboucas this is now done, the review also got reset.